great_expectations/checkpoint/checkpoint.py

```diff
@@ -911,7 +917,7 @@ def construct_from_config_args(
         config_version: Optional[Union[int, float]] = None,
         template_name: Optional[str] = None,
         module_name: Optional[str] = None,
-        class_name: Optional[str] = None,
+        class_name: Literal["Checkpoint", "SimpleCheckpoint"] = "Checkpoint",
```

Comment:
```suggestion
        class_name: str = "Checkpoint",
```
This causes warnings in `data_context` because the type there is just a `str`